src/Native/Runtime/inc/eetype.inl

```diff
@@ -669,6 +704,22 @@ inline EEType * EEType::GetArrayBaseType()
     if ((get_RareFlags() & HasDynamicallyAllocatedDispatchMapFlag) != 0)
         cbOffset += sizeof(UIntTarget);
 
+    if (eField == ETF_GenericDefinition)
+    {
+        ASSERT(IsGeneric());
+        return cbOffset;
+    }
+    if (IsGeneric())
+        cbOffset += sizeof(UIntTarget);
+
+    if (eField == ETF_GenericComposition)
+    {
+        ASSERT(IsGeneric());
+        return cbOffset;
+    }
+    if (IsGeneric())
```

Comment:
> Do open generics have the IsGeneric flag set

Nope, the only bits the binder sets are: `EEType::GenericTypeDefEEType`, `EEType::ValueTypeFlag`, and `EEType::IsInterfaceFlag`.